pakwheels_clone/cars/management/commands/extract_json_data.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+from cars.models import CarDetails, CarModel, CarManufacturer, CarFeatures, CarImages
+from users.models import User
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument("json_file", type=str)
+
+    def handle(self, *args, **kwargs):
+        json_file = kwargs['json_file']
+        file = open(json_file,'r')
+        used_car_data = json.load(file)
+        for car_listed in used_car_data:
+          self.setup_car_details(car_listed)
+
+    def setup_car_details(self, car_listed):
+        model, _ = CarModel.objects.get_or_create(car_model=car_listed['vehicle_model'])
+        manufacturer, _ = CarManufacturer.objects.get_or_create(car_make=car_listed['vehicle_manufacturer'])
+        user, _ = User.objects.get_or_create(name=car_listed['seller_name'])
+        vehicle_information = car_listed['vehicle_information']
+
+        car = CarDetails.objects.create(
```

Comment:
use factory boy for creating initial data or loading it https://factoryboy.readthedocs.io/en/stable/orms.html